docs/index.html

```diff
@@ -18,16 +18,16 @@
 <body>
     <header>
         <div class="logo">
-            DSS
-            <div class="logo-text">dual-side scroll
+            SP
+            <div class="logo-text">scroll progress
                 <div class="version">v1.0.1</div>
```

Comment:
```suggestion
                <div class="version">v1.0.1</div>
```
Надо исправить номер версии